### ISSUE TITLE
[Bug]: Create index hang after bulkinsert

### ISSUE THREAD

**Issue by wayblink** [milvus-io/milvus#21979]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:master
- Deployment mode(standalone or cluster):
- MQ type(rocksmq, pulsar or kafka):    
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Create index after bulkinsert will hang. It should be done in a few seconds. After couple of minutes, create index can work.

It looks because  we build segment index with CreateTime inherited from segment's LastExpireTime. In bulkInsert，LastExpireTime is set current Ts + ImportTaskExpiration. 

![image](https://user-images.githubusercontent.com/18096561/216593248-3a530fc5-c40d-4c02-8870-5d3ded79ce41.png)
![image](https://user-images.githubusercontent.com/18096561/216593932-583e0e2d-b638-44d4-8c14-292b45249b02.png)

**Comment by czs007** [comment]
“ In bulkInsert，LastExpireTime is set current Ts + ImportTaskExpiration.”
What is the purpose of this behavior? @wayblink

**Comment by xiaofan-luan** [comment]
> “ In bulkInsert，LastExpireTime is set current Ts + ImportTaskExpiration.” What is the purpose of this behavior? @wayblink

This is introduced by another PR which I thought bulk inserted segment expire time is not useful.
Now  bulk inserted segment expired is used when bulk insert failed and datacoord gc the segment.

**Comment by xiaofan-luan** [comment]
Maybe we should introduce another parameter for gc failed bulk insert segment to avoid misunderstanding?

**Comment by wayblink** [comment]
I think the inappropriate point is we set segment index's createTime as segment's LastExpireTime. This make bulkinsert and create index messing up. Segment index's createTime should be allocated in real time, I tried fix in #21980, it works. One thing I need to verify, bulkinsert segments shouldn't be created index before bulkinsert task done.

**Comment by wayblink** [comment]
fixed in #22396
